uiowa/uiowa
# Feature: Events summary

Resolves https://github.com/uiowa/uiowa/issues/5503

# How to test
- Review files changed.
- ` ddev blt ds --site=sandbox.uiowa.edu`
- Sync a site and change the sitenow_events endpoint to dev `drush @sandbox.local cset sitenow_events.settings base_endpoint 'https://content.dev.drupal.uiowa.edu/api/v1/'`.
- Go to https://sandbox.uiowa.ddev.site/digital-signage-events and look for events that are using the new summary field. 
- Verify that the description field will work if there is no summary field.
